HoloJS/HoloJsHost/ScriptsLoader.h

```diff
@@ -11,7 +16,7 @@ namespace HologramJS
 		concurrency::task<bool> LoadScripts(Windows::Storage::StorageFolder^ root, const std::wstring& jsonPath);
 		concurrency::task<bool> DownloadScripts(const std::wstring& uri);
 
-		const std::list<std::wstring>& LoadedScripts() { return m_loadedScripts; }
+		const std::list<Script*>& LoadedScripts() { return m_loadedScripts; }
```

Comment:
This should be a unique or shared pointer to automatically free the allocated memory.